release-team/role-handbooks/enhancements/README.md

```diff
@@ -83,28 +83,25 @@ It is important that this process be followed and documentation remain up-to-dat
 #### Pre-Collection (Week 0)
 
 - Duplicate the previous enhancement collection spreadsheet into your own Google Drive and adjust it for the current milestone. Enhancements Tracking sheet is shortlinked with the pattern `k8sxyy-enhancements` e.g., http://bit.ly/k8s113-enhancements. Update the permissions on the sheet via the _Share_ settings available in the top right of the sheet: Enable anyone with the link to _view_ the sheet. Next, grant _Edit_ access to yourself, prior lead, release lead, shadows, and the [Kubernetes Release Team Google Group][rt-group] . Lastly, add _Comment_ access for the [SIG Release Google Group][sig-release-group], [SIG Docs Google Group][sig-docs-group], and [SIG Leads Google Group][sig-leads-group]. When complete, remove all currently tracked issues from both the `Enhancements` and `Docs` tabs.
-- Within the tracking sheet, filter the `data` tab to remove Issues that were completed in a previous release. Copy the `Issue Number` column to the `Enhancements` tab. The tab will auto-populate with a portion of the other relevant data. Once complete, set their `Enhancement Status` to `Pending Inclusion`. This will serve as the initial seed of Enhancements that need to be evaluated for the release.
-- Examine the `Removed From Milestone` tab and visit those issues to see if they have been added to the current milestone. These enhancements likely slipped because of docs or testing. Ping owners to make sure they are targeted for the current milestone. When complete, clear the issues from the tab.
 - Find Issues from previous milestone that have graduated to Stable. Remove `tracked/yes` or `tracked/no` labels. Check to see if the KEP status has been updated to `implemented`. If it has, close the issue. If it has not, ask the issue contact to both update the KEP status field and close the Enhancement issue once the update PR has merged.
 - Clear out previous milestone issues. Ping owners to get them to drop milestone or add to the current milestone so it can be tracked. This can be a responsibility for shadow roles.
```

Comment:
I think we need to change or remove the last two sentences in this bullet. Lead can clear the previous milestone but new milestone won't be added until the sigs put it in their list

Reply:
I removed it -- I believe it's covered by the line below "Close previous milestone by ensuring that there are no open issues/PRs in that milestone."